src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -99,15 +100,7 @@ private static ImmutableArray<string> GetRazorSemanticTokenTypes()
         return builder.ToImmutable();
     }
 
-    private static readonly string[] s_tokenModifiers = new string[] {
-        // TODO: These two should come from Roslyn once an available version is inserted
-        // Razor
-        "static",
-        // C# Modifiers
-        "ReassignedVariable",
-        // Razor background
-        "razorCode" // Not using nameof() because the convention is for modifiers to start with a lowercase letter. Yes I am aware of what the line above this says.
-    };
+    private static readonly string[] s_tokenModifiers = RazorSemanticTokensAccessor.GetTokenModifiers().Concat(Enum.GetNames(typeof(RazorTokenModifiers))).ToArray();
```

Comment:
Can we make this an immutable array?

Reply:
We'd have to immediately ToArray it in order to put it in the protocol type, so given its a static initalizer that will be called once, I figured its not worth it. If you strongly disagree, let me know.

Reply:
Ah I forgot about protocol types being arrays 😢